core/settings.py

```diff
@@ -39,6 +41,7 @@
     "django.contrib.staticfiles",
     "bid",
     "corsheaders",
+    "rest_framework",
```

Comment:
after installing `rest_framework` locally in venv, make sure to add the module to `requirements.txt` so that it can work for others.
use `pip install -r requirements.txt`.